nvme.c

```diff
@@ -2015,7 +2015,7 @@ static int get_phy_rx_eom_log(int argc, char **argv, struct command *cmd,
 		phy_rx_eom_log_len = le16_to_cpu(phy_rx_eom_log->hsize);
 	}
 
-	phy_rx_eom_log = realloc(&phy_rx_eom_log, phy_rx_eom_log_len);
+	phy_rx_eom_log = realloc(phy_rx_eom_log, phy_rx_eom_log_len);
```

Comment:
This should be part of the first patch. Though we don't have a `nvme_realloc` yet. Just introduce a simple `nvme_realloc` function which does the alloc new block if necessary things. Doesn't need to be fancy, correctness is more important than premature optimization.

Reply:
Pulled this change into the other commit and added a `nvme_realloc` function.